pyproject.toml

```diff
@@ -15,11 +15,11 @@ dependencies = [
     "magicgui>=0.9.1",
     "siphash>=0.0.1",
     "siphash24>=1.7",
+    "scyjava>=1.10.0",
+    "bioio-bioformats>=1.0.0",
+    "torch>=2.5.1",
```

Comment:
Feels like `black` should be a dev dependency?

Reply:
@kmilo9999 Consider also adding ruff to the dev dependency group. Besides formatting, it also has a good linter.

Reply:
Done